src/app.ts

```diff
@@ -0,0 +1,14 @@
+
+import express from 'express';
+import loginRouter from './routes/loginRouter';
+import signUpRouter from './routes/signUpRouter';
+
+const app = express();
+const port = 3000;
+
```

Comment:
We should check the versioning [here](https://www.freecodecamp.org/news/how-to-version-a-rest-api/).